lib/galaxy/objectstore/irods.py

```diff
@@ -81,6 +82,7 @@ def parse_config_xml(config_xml):
             _config_xml_error("cache")
         cache_size = float(c_xml[0].get("size", -1))
         staging_path = c_xml[0].get("path", None)
+        cache_updated_data = string_as_bool(c_xml.get("cache_updated_data", "True"))
```

Comment:
typo: missed `[0]`
```suggestion
        cache_updated_data = string_as_bool(c_xml[0].get("cache_updated_data", "True"))
```

Reply:
this will take care of one unit test failure.

Reply:
Thanks, John! I have to confess - I use your GitHub infra to test most of the stuff since on my Mac it is so slow and lots of tests fail anyway just because it is Mac (or I did not configure something correctly :))

Reply:
No worries! It does take a lot of time locally.